codex/codex.nim

```diff
@@ -118,7 +118,7 @@ proc new*(T: type CodexServer, config: CodexConf): T =
 
     wallet = WalletRef.new(EthPrivateKey.random())
     network = BlockExcNetwork.new(switch)
-    localStore = FSStore.new(config.dataDir / "repo", cache = cache)
+    localStore = SQLiteStore.new(config.dataDir / "repo", cache = cache)
```

Comment:
Maybe we should allow making this parametrizable and be able to select FS vs SQL, but we don't have to do this as part of this PR.

Reply:
Tracking in #179.